src/SignhostAPIClient/Rest/DataObjects/Transaction.cs

```diff
@@ -7,9 +7,11 @@ public class Transaction
 	{
 		public string Id { get; set; }
 
+		public IDictionary<string, FileEntry> Files { get; set; }
+
 		public int Status { get; set; }
 
-		public bool Seal { get; set; }
+		public bool? Seal { get; set; }
```

Comment:
Perhaps changing it to `public bool Seal { get; set; } = true` would be more appropriate? This way the default value as stated in the documentations will still be used if Seal is not implicity set by the user.